src/types.h

```diff
@@ -628,8 +628,8 @@ namespace librealsense
     inline float3x3 transpose(const float3x3 & a) { return{ {a.x.x,a.y.x,a.z.x}, {a.x.y,a.y.y,a.z.y}, {a.x.z,a.y.z,a.z.z} }; }
     inline bool operator == (const pose & a, const pose & b) { return a.orientation == b.orientation && a.position == b.position; }
     inline float3 operator * (const pose & a, const float3 & b) { return a.orientation * b + a.position; }
-    inline pose operator * (const pose & a, const pose & b) { return{ a.orientation * b.orientation, a * b.position }; }
-    inline pose inverse(const pose & a) { auto inv = transpose(a.orientation); return{ inv, inv * a.position * -1 }; }
+    inline pose operator * (const pose & a, const pose & b) { return{ a.orientation * b.orientation, a.position + b.position }; }
```

Comment:
Please recheck - the output position requires Matrix*Vector multiplication.
https://www.brainvoyager.com/bv/doc/UsersGuide/CoordsAndTransforms/SpatialTransformationMatrices.html